inst/extdata/mvp/product-carbon-transition-risk_review.Rmd

```diff
@@ -0,0 +1,308 @@
+
+``` {r}
+# percentile of all products: 'perc_all'
+data_short <- data_short %>%
+  mutate(perc_all = rank(production_emissions) / length(production_emissions))
+
+# percentile of all products with same unit: 'perc_unit'
+data_short <- data_short %>%
+  group_by(unit) %>%
+  mutate(perc_unit = rank(production_emissions) / length(production_emissions)) %>%
+  ungroup()
+
+# percentile of all products with same sector 'ISIC Classification' and 'Unit':
+# 'perc_unit_isic'
+data_short <- data_short %>%
+  group_by(unit, isic_class) %>%
+  mutate(perc_unit_isic = rank(production_emissions) / length(production_emissions)) %>%
+  ungroup()
+```
+
+## 2. Assign scores to the activities based on their position within the distribution
+
+``` {r}
+# assign scores to percentiles
+data_short <- data_short %>%
+  mutate(score_all = case_when(
+    perc_all < 0.3 ~ "low",
+    perc_all >= 0.3 & perc_all < 0.7 ~ "medium",
+    perc_all >= 0.7 ~ "high",
+  ))
+
+data_short <- data_short %>%
+  mutate(score_unit = case_when(
+    perc_unit < 0.33 ~ "low",
+    perc_unit >= 0.3 & perc_unit < 0.7 ~ "medium",
+    perc_unit >= 0.7 ~ "high",
+  ))
+
+data_short <- data_short %>%
+  mutate(score_unit_isic = case_when(
+    perc_unit_isic < 0.3 ~ "low",
+    perc_unit_isic >= 0.3 & perc_unit_isic < 0.7 ~ "medium",
+    perc_unit_isic >= 0.7 ~ "high",
+  ))
+
+# show distributions (each score should have same probability for score_all)
+barplot(prop.table(table(data_short$score_all)))
+barplot(prop.table(table(data_short$score_unit)))
+barplot(prop.table(table(data_short$score_unit_isic)))
+```
+
+### Step 2: Aggregating on the company-level
+
+      Aggregating on the company-level the percentage of transistion risk products for each company.
+      Calculate 'Product Carbon TR Indicator' for example companies from Hanish's compiled dataset
+
+## Data
+
+## 1. Loading sample company data set & create sample product list
+
+    NOTE: 
+
+    - Problem A with dataset: in the MVP_company_sample dataset the activity names from ecoinvent have an underscore "\_" instead
+      of space in their name. E.g., there is an activity called "treatment_of_tyre_wear_emissions_passenger_car" instead of            "treatment of tyre wear emissions, passenger car" as in the original dataset.
+
+    - Quick solution for now (I don't want to invest time in replacing underscores with spaces and vice versa: I replace the            products with random example from ecoinvent
+
+    - Problem B with dataset: in the sample MVP dataset, some company_name were associated with multiple company_ids (example:          instead of ID number 00001 for each row, excel filled in 00002, 00003, 00004 in the following rows). I adjusted that            manually, i.e. the issue is resolved in the following dataset.
+
+``` {r}
+# load sample data
+data_companies <- params$input2 %||% 
+  read_csv(pctr_path("input_data", "MVP_sample_dataset.csv"))
```

Comment:
Again, this weird code allows me to inject data into `pctr_company_scores()`. Don't worry for now. If you have the data in the right place (`"~/Downloads/tilt/...") it should just work.